libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/TypeRep.hs

```diff
@@ -1123,6 +1130,7 @@ hasOrd rep xx = explain ("'hasOrd " ++ show rep ++ "' fails") (help rep xx)
     help BoolR v = pure $ With v
     help DRepR v = pure $ With v
     help (PoolMetadataR _) v = pure $ With v
+    help DRepStateR _ = failT ["DRepState does not have Ord instance"]
```

Comment:
Just a question. Why can't we add an Ord instance for DRepState?

Seems like Anchor can easily get an Ord instance

Reply:
I don't think it's impossible, I just didn't want to bother with it :sweat_smile:

Reply:
We can add it in a different PR

Reply:
Postponed in #3624